src/Http/ServerRequestFactory.php

```diff
@@ -54,6 +54,7 @@ public static function fromGlobals(
             'webroot' => $uri->webroot,
             'base' => $uri->base,
             'session' => $session,
+            'mergeFilesAsObjects' => Configure::read('ServerRequest.mergeFilesAsObjects'),
```

Comment:
This configuration flag could go under `App` so that we don't need to add another top level key.

Reply:
Also the flag could be named `uploadedFilesAsObjects`. The merging is an internal detail and doesn't need to reflect in flag name.

Reply:
If the flag is renamed, shouldn't also the config key be the same renamed one for consistency?

Reply:
Maybe... maybe not. The class kinda already has uploads as objects, as all uploads are available as objects via `getUploadedFiles()`.